test/e2e/tests/request-queuing/dapp1-send-dapp2-signTypedData.spec.js

```diff
@@ -58,24 +57,13 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
         await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
         await driver.clickElement({
```

Comment:
```suggestion
        await driver.clickElementAndWaitForWindowToClose({
```
We should wait until the dialog is closed before opening the new dapp url, otherwise the dapp will never load, see ci failure:

![Screenshot from 2024-10-08 12-14-26](https://github.com/user-attachments/assets/0f495969-8c7b-4f86-b866-9599ff5204b9)

Repro locally too

https://github.com/user-attachments/assets/949d831e-e115-4bd9-ad34-b7ab05ddb0c4